### ISSUE TITLE
[Support CSV] Create import flow for tags

### ISSUE THREAD

**Issue by rlinoz** [Expensify/App#47870]
Complete [this section of the doc](https://docs.google.com/document/d/1YfARkp3Zko6hDjS1EnO84gy7gGmoRK712ZJf-wmDNRw/edit#bookmark=id.nghzwfmmmdgr) for members.

Let's rely on this PR for reference https://github.com/Expensify/App/pull/47827

<details><summary>Issue Owner</summary>Current Issue Owner: @CortneyOfstad</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @rushatgabhane</details>

**Comment by filip-solecki** [comment]
Hi! I am Filip from SWM and I'd like to work on this!

**Comment by Guccio163** [comment]
Hi, I'm Wiktor Gut from SWM agency, I'm stepping up for Filip since he's OOO; I'll link here PR handling both importing and exporting `Tags` spreadsheets

**Comment by MonilBhavsar** [comment]
Looks like PR was deployed to production two weeks ago and we can close this issue

**Comment by MonilBhavsar** [comment]
@rlinoz do we need to pay C+ here or payment is being handled elsewhere? cc @CortneyOfstad

**Comment by rlinoz** [comment]
Ah sorry I missed this, but yeah, I get confused by:
> there is a project based payment for this

**Comment by mountiny** [comment]
Yeah I think we pay the c+ who helped with the project all in one payment at the end of the project

**Comment by mountiny** [comment]
@rlinoz should create a payment issue to handle it with the details/ amount. Can you please create one and assign the bug zero person using Bug label for payment? Thanks!

**Comment by rlinoz** [comment]
Sorry for the inconvenience, I am going to handle this

**Comment by mallenexpensify** [comment]
@rlinoz I can help too! @rlinoz , wanna chat in NewDot 1:1 to get aligned?

**Comment by CortneyOfstad** [comment]
Hey @mallenexpensify @rlinoz — I am finally back in office after PTO and conferences, and I apologize for the delay here! Where are we at in regards to this? Thanks!

**Comment by mallenexpensify** [comment]
@rushatgabhane did you mention that $250 per PR would work for payment here? If so, wanna link all the PRs and we'll review then approve the total?  Then, finally, get ya paid.

**Comment by mallenexpensify** [comment]
Contributor+: @rushatgabhane due $250 via NewDot 

Chatted with Rushat and $250/per PR works. 

@rushatgabhane , wanna link any other issues that are due payment and I can help get 'em sorted?  Tag me in any comments if so

**Comment by CortneyOfstad** [comment]
@rushatgabhane @MonilBhavsar @Guccio163 — any update on this?

**Comment by MonilBhavsar** [comment]
Looks like this waiting for @rushatgabhane. Can we close this?

**Comment by CortneyOfstad** [comment]
@rushatgabhane ^^^ bump — thanks!

**Comment by rushatgabhane** [comment]
- https://github.com/Expensify/App/issues/48587
- https://github.com/Expensify/App/pull/48687
- https://github.com/Expensify/App/pull/48876
- https://github.com/Expensify/App/pull/47827
- https://github.com/Expensify/App/pull/45614
- https://github.com/Expensify/App/pull/45593
- https://github.com/Expensify/App/pull/44820
- https://github.com/Expensify/App/pull/48877
- https://github.com/Expensify/App/pull/48906
- https://github.com/Expensify/App/pull/48609
- https://github.com/Expensify/App/pull/48539
- https://github.com/Expensify/App/pull/48556
- https://github.com/Expensify/App/pull/48569

**Comment by rushatgabhane** [comment]
thank you for confirming!

so i guess we have payment due for 6 issues then 😃

#48587
#48876
#47827
#48877
#48539
#48569

**Comment by CortneyOfstad** [comment]
Tackling the payments now!

**Comment by CortneyOfstad** [comment]
@rushatgabhane If anything else is needed for the payment request via NewDot, just let me know and I can adjust the summary as needed. Thanks!

**Comment by garrettmknight** [comment]
https://github.com/Expensify/App/issues/57554#issuecomment-2729192417

**Comment by garrettmknight** [comment]
$250 approved for @rushatgabhane
